WSL/tutorials/wsl-database.md

```diff
@@ -111,21 +111,32 @@ To work with with PostgreSQL databases in VS Code, try the [PostgreSQL extension
 
 ## Install MongoDB
 
-To install MongoDB on WSL (Ubuntu 18.04):
+To install MongoDB (version 5.0) on WSL (Ubuntu 20.04):
 
-1. Open your WSL terminal (ie. Ubuntu 18.04).
+1. Open your WSL terminal (ie. Ubuntu) and go to the root directory: `cd ~`
 2. Update your Ubuntu packages: `sudo apt update`
-3. Once the packages have updated, install MongoDB with: `sudo apt-get install mongodb`
-4. Confirm installation and get the version number: `mongod --version`
+3. Import the public key used by the MongoDB package management system: `wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | sudo apt-key add -`
+4. Create a list file for MongoDB: `echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list`
+5. Reload local package database: `sudo apt-get update`
+6. Install MongoDB packages: `sudo apt-get install -y mongodb-org`
+7. Confirm installation and get the version number: `mongod --version`
+8. Go to your home directory: `cd / && cd home`
+9. Make a directory to store data: `mkdir -p data/db`
+10. Run a Mongo instance: `sudo mongod --dbpath ~/data/db`
+11. Check to see that your MongoDB instance is running with: `ps -e | grep 'mongod'`
+12. To exit the MongoDB Shell, use the shortcut keys: Ctrl + C
+
+> [!TIP]
+> Installing MongoDB may require slightly different steps depending on the Linux distribution being used for installation. See the [MongoDB installation tutorials](https://docs.mongodb.com/manual/installation/#mongodb-installation-tutorials). Also note that MongoDB installation may differ depending on the version # that you are aiming to install. Use the version drop-down list in the top-left corner of the MongoDB documentation to select the version that aligns with your goal.
 
 There are 3 commands you need to know once MongoDB is installed:
 
 - `sudo service mongodb status` for checking the status of your database.
```

Comment:
These commands will error out in WSL. I don't think we should include them.

Reply:
The edits I made to warning below explain why and offer a way for users to get set up.